lib/github/kv.rb

```diff
@@ -71,6 +71,7 @@ def initialize(encapsulated_errors = [SystemCallError], use_local_time: false, &
       @encapsulated_errors = encapsulated_errors
       @use_local_time = use_local_time
       @conn_block = conn_block
+      @table_name = GitHub::SQL::LITERAL(GitHub::DS.config.table_name).value
```

Comment:
I generally like this pattern for config and I've recently used it for the flipper gem. As I look at it though, the configuration for this class now feels like it is in two places -- initialize arguments and a config global. I'm curious if you have any thoughts on one of the following options:

* moving the other initialize options to config as well and changing initialize to receiving only config (with default to GitHub::DS.config)
* simply adding `table_name` as a parameter for `initialize` and dropping the new config

Reply:
I want to keep `table_name` in a config as it's used in other places outside of `GitHub::KV` so I went with option 1 😄